docs/Authorization.md

```diff
@@ -248,7 +248,7 @@ const MyPage = ({ permissions }) => (
         }
     </Card>
 )
-const MyPageWithPermissions = ({ location, match }) => (
+const MyPageWithPermissions = ({ location, match }, route) => (
```

Comment:
nope, functional components only accept one parameter.

Reply:
The function inside is referencing a route parameter which does not exist though. Where does that come from?